h/static/scripts/directive/test/group-list-test.js

```diff
@@ -122,20 +102,9 @@ describe('groupList', function () {
     var shareLinks = element.find('.share-link-container');
     assert.equal(shareLinks.length, 1);
 
-    var linkField = element.find('.share-link-field');
-    assert.equal(linkField.length, 1);
-    assert.equal(linkField[0].value, GROUP_LINK);
-  });
-
-  it('should toggle share link on click', function () {
-    var element = createGroupList();
-    var toggleLink = element.find('.share-link-toggle');
-    var expander = element.find('.share-link-expander');
-    assert.ok(isElementHidden(expander));
-    toggleLink.click();
-    assert.ok(!isElementHidden(expander));
-    toggleLink.click();
-    assert.ok(isElementHidden(expander));
+    var link = element.find('.share-link');
+    assert.equal(link.length, 1);
+    assert.equal(link[0], GROUP_LINK);
```

Comment:
This is a neat trick... I was looking at this thinking "How on earth can that possibly pass? `link[0]` is a DOM Element, and `GROUP_LINK` is a string...!?"

Well, it turns out `String(anchor_element)` coerces to the `href` of the element.

While this does indeed allow this test to pass, I think it might be good to be a bit more explicit with what you're testing here.